CloudCollector/src/main/java/com/capitalone/dashboard/model/AWSConfig.java

```diff
@@ -0,0 +1,36 @@
+package com.capitalone.dashboard.model;
+
+public class AWSConfig extends CollectorItem {
```

Comment:
should just use: http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html#DefaultAWSCredentialsProviderChain()

So that it will use instance creds or the standard locations to match aws cli